packages/@react-spectrum/s2/src/SegmentedControl.tsx

```diff
@@ -32,16 +32,22 @@ export interface SegmentedControlProps extends Omit<RadioGroupProps, 'isReadOnly
    * Whether the segmented control is disabled.
    */
   isDisabled?: boolean,
-  /**
-   * Defines a string value that labels the current element.
-   */
-  'aria-label': string
+  /** The id of the currently selected item (controlled). */
+  selectedKey?: string | null,
```

Comment:
Kinda weird that these have to be strings and not numbers like our other selection APIs, because RadioGroup only accepts strings in the `value` prop, but it is closer.

Reply:
[fix: add number type in value](https://github.com/adobe/react-spectrum/pull/6636#top)

Reply:
looks like a test works for it